ddtrace/contrib/django/middleware.py

```diff
@@ -56,6 +56,8 @@ def process_response(self, request, response):
             span = _get_req_span(request)
             if span:
                 span.set_tag(http.STATUS_CODE, response.status_code)
+                if response.status_code == 500:  # Some exceptions are not handled by `process_exception`
+                    span.set_traceback()
```

Comment:
Is that attaching an interesting traceback? If the caught exception then when thought other middlewares, maybe we lost track of the original issue?
Let's figure it out with examples.